src/routes/docs/products/databases/queries/+page.markdoc

```diff
@@ -165,6 +170,11 @@ Query::select(["name", "title"])
 ```swift
 Query.select(["name", "title"])
 ```
+```http
+GET /v1/databases/<DATABASE_ID>/collections/<COLLECTION_ID>/documents?queries[]={"method":"select","values":["name","title"]} HTTP/1.1
+Content-Type: application/json
+X-Appwrite-Project: <PROJECT_ID>
```

Comment:
Since the other code examples have just the `Query.select()` part, maybe it would be simpler and clearer if we just put the JSON here?

```suggestion
{"method":"select","values":["name","title"]}
```

Honestly, i'm really hoping AI can scrape this and then learn how to properly build queries for Appwrite 😅

Reply:
Yes, that makes sense. I've updated the HTTP examples